command/login.go

```diff
@@ -50,7 +51,12 @@ Login Options:
 
   -oidc-callback-addr
     The address to use for the local OIDC callback server. This should be given
-    in the form of <IP>:<PORT> and defaults to "localhost:4649".
+    in the form of <IP>:<PORT> and defaults to "localhost:4649". It is only
+    required if using OIDC auth method type. 
+
+  -login-token
```

Comment:
```suggestion
  -login-token=<token>
```

We usually document flags that take values to differentiate them from boolean types (like `-json` below), but I don't know how consistent we are about it 😅

Reply:
eh we don't do this for any of the auth method commands...